packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerFeeWallet.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+
+/* Library Imports */
+import { Lib_PredeployAddresses } from "../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { OVM_ETH } from "../predeploys/OVM_ETH.sol";
+
+/**
+ * @title OVM_SequencerFeeWallet
+ * @dev Simple holding contract for fees paid to the Sequencer. Likely to be replaced in the future
+ * but "good enough for now".
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_SequencerFeeWallet {
+
+    /*************
+     * Constants *
+     *************/
+
+    uint256 constant MIN_WITHDRAWAL_AMOUNT = 10 ether;
+    address constant L1_FEE_WALLET = 0x1111111111111111111111111111111111111111;
+
+
+    /********************
+     * Public Functions *
+     ********************/
+
+    function withdraw(
+        uint256 _amount
+    )
+        public
+    {
+        require(
+            _amount >= MIN_WITHDRAWAL_AMOUNT,
```

Comment:
Why this require? Seems like adding a headache

Reply:
Ah so this is actually a very smart `require` statement because it prevents a bad greif where someone repeatedly calls withdraw for ETH amounts that is less than the cost of relaying the withdrawal. It could let someone basically just burn all of our ETH for pretty small transaction fees.

Reply:
That said, @smartcontracts  what do you think about removing `_amount` and just always withdraw the full amount of Sequencer fees. Am I right that there's no reason to let the amount be configurable?

Reply:
Good point, I'll remove the amount.